Add new section on website for links to blog posts, podcasts and talks. (#2216)
grafana/mimir: docs/sources/talks-and-articles/_index.md

---

> title: "Further reading"

Reviewer: ```suggestion
title: "Reference: Learning resources"
```

> menuTitle: "Further reading"

Reviewer: ```suggestion
menuTitle: "Learning resources"
```

> description: "Talks and articles about Grafana Mimir"

Reviewer: ```suggestion
description: "Blog posts, podcasts, and videos about Grafana Mimir"
```

weight: 40
keywords:

>   - Grafana Mimir talks and articles

Reviewer: ```suggestion
  - Grafana Mimir blog posts, podcasts, and videos
```
@brendamuir, how are we standardizing on keywords in metadata?

Reviewer: Please input my suggestion and if these need to change, we can change them in a subsequent PR.

Author: This is already in.

---

> # Furhter reading

Reviewer: ```suggestion
# Further reading
```

Reviewer: Please move this to a `Reference: Learning resources` section parallel to the other references in the operator’s guide.

> To learn more about Grafana Mimir, consul the following talks and articles.

Reviewer 2: ```suggestion
To learn more about Grafana Mimir, consult the following talks and articles.
```

> To learn more about Grafana Mimir, consul the following talks and articles.

Reviewer: ```suggestion
To learn more about Grafana Mimir, see the following resources:
```

> - March 2022 blog post "[Announcing Grafana Mimir, the most scalable open source TSDB in the world](https://grafana.com/blog/2022/03/30/announcing-grafana-mimir/)"

Reviewer 2: What do you think about an anti-chronological order? Most recent, first.

Author: Good idea.

- April 2022 blog post: "[How we scaled our new Prometheus TSDB Grafana Mimir to 1 billion active series](https://grafana.com/blog/2022/04/08/how-we-scaled-our-new-prometheus-tsdb-grafana-mimir-to-1-billion-active-series/)"
- April 2022 video tutorial: "[How to migrate to Grafana Mimir in less than 4 minutes](https://grafana.com/blog/2022/04/25/video-how-to-migrate-to-grafana-mimir-in-less-than-4-minutes/)"
- April 2022 blog post: "[How Grafana Mimir’s split-and-merge compactor enables scaling metrics to 1 billion active series](https://grafana.com/blog/2022/04/19/how-grafana-mimirs-split-and-merge-compactor-enables-scaling-metrics-to-1-billion-active-series/)"
- May 2022 blog post: "[Scaling Grafana Mimir to 500 million active series on customer infrastructure with Grafana Enterprise Metrics](https://grafana.com/blog/2022/05/24/scaling-grafana-mimir-to-500-million-active-series-on-customer-infrastructure-with-grafana-enterprise-metrics/)"
- May 2022 podcast episode: [Grafana's Big Tent podcast: Grafana Mimir: Maintainers tell all](https://bigtent.fm/4), [transcript](https://grafana.com/blog/2022/05/03/grafana-mimir-maintainers-tell-all/)

> - June 2022 GrafanaCon 2022 talk: "[For billion-series scale or home IoT projects, get started in minutes with Grafana Mimir (using Helm)](https://grafana.com/go/grafanaconline/2022/grafana-mimir-migrate-your-metrics-in-minutes/)"

Author: I'm not too happy with this suggestion... Given how few links we have, do we really need categories? They will also break the chronological ordering.
